src/app/components/Curation/index.tsx

```diff
@@ -74,13 +82,21 @@ const Curation = ({
         />
       ) : null;
     case MOST_READ:
+      console.log('in case MOST_READ', mostRead);
       return (
         <MostRead
           data={mostRead}
           columnLayout="twoColumn"
           headingBackgroundColour={GHOST}
         />
       );
+    case RADIO_SCHEDULE:
+  //     initialData,
+  // radioScheduleEndpointOverride,
+  // lang,
+  // className,
+      console.log('in case RADIO_SCHEDULE', radioSchedule);
+      return radioSchedule && <RadioScheduleContainer initialData={radioSchedule} />
```

Comment:
I don't think we need to include this additional check for whether radio schedule is defined - we've taken care of this in the BFF, so that if a radio schedule exists, you'll get the data, otherwise you won't, and the `getComponentName` check will never return RADIO_SCHEDULE

Also, a check for the existence of the data is carried out within the RadioScheduleContainer, and if the radio schedule object is undefined, it will go and fetch it client-side.
```suggestion
      return <RadioScheduleContainer initialData={radioSchedule} />
```

Reply:
I thought it was pointless but in our typescript tangle it appeared to help so I just put it there to get TS to shut up 😂 Now we have the real fixes for that though, this is definitely unnecessary!